docs/modules/operation/pages/deep-dive/flows/thresholding.adoc

```diff
@@ -86,11 +89,47 @@ You can set the following properties:
 | Default
 
 | flowThreshold.stepSizeMs
-| Interval for checking thresholds and doing data collection in milliseconds.
+| Interval for checking thresholds and doing data collection, in milliseconds.
 | 300000
 
 | flowThreshold.idleTimeoutMs
 | Timeout for cleaning up idle sessions.
 | 900000
 
 |===
+
+== Application threshold behavior
+
+This section describes how flows thresholds work in specific scenarios.
+
+=== Undefined application
+
+If a flow threshold does not specify an application, you will see the following behavior:
+
+* If the `filterOperator` is set to OR, there must be at least one filter or the threshold configuration will not match.
+
+* If the `filterOperator` is set to AND, an empty list of filters will result in a match.
+
+=== Apply flow threshold to all applications
+
+You may want to create a flow threshold that applies to all applications; for example, "if any application consumes more than X bandwidth ...".
+
+To apply a flow threshold to all applications, use a `match-all` regex expression like `.*` inside the filter.
+
+//not sure what was meant in the Jira by "But then you could just threshold on the interface statistics."
```

Comment:
I've read part of the question as "how can we threshold over the sum of all applications". Which is the whole interface utilization and other sums are not possible for now.